_episodes/licenses.md

```diff
@@ -1,24 +1,149 @@
+- "Apply a licence to your data"
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "A permissive licence ensures the re-usability of your data."
+- "Many big inter-comparison projects already have suitable licences."
+- "For derived work, existing licences may restrict your choice of licence."
+- "Ownership of data (FIXME)"
 ---
 
-
 General info on licenses
+------------------------
+
+Let's get started with a small exercise.
+
+> ## Choosing a licence
+> Go to <https://chooser-beta.creativecommons.org/> and follow the steps.
+>
+>- What licence did you arrive at?
+>- Does the licence fit with the FAIR principles?
+>- Discuss.
+{: .challenge}
+
+One of the key parts of FAIR data is that they are **reusable** not only practically, but also legally. By attaching a licence to your work, you specify who can reuse your data, for which purposes, and what should be done with derived work. Creative Commons (CC) are the most used licence in research. They are widely recognized, easy to apply, and juridically sound. The CC licence is built on four cornerstones:
+
+- Attribution (BY): Credit must be given to you, the creator.
+- Share-Alike (SA): Adaptations must be shared under the same terms.
+- Non-Commercial (NC): Only noncommercial use of your work is permitted.
+- Non-Derivative (ND): No derivatives or adaptations of your work are permitted.
+
+The six CC licences are combinations of these cornerstones.
+
+> ## Note
+> Without a licence you keep the copyright to yourself. This prohibits anyone from using the data for their own work.
+{: .callout}
+
+These are some points to consider when choosing a licence:
+
+- A licence cannot be revoked once the data has been shared.
+- As the origial author, you can change the licence for future publications.
+- Sensivite data / embargo
+- The less restrictions you add to your data, the more it can be re-used
+
+Further reading:
+- <https://www.dcc.ac.uk/guidance/how-guides/license-research-data>
+- <https://www.openaire.eu/research-data-how-to-license/>
+
+> ## Excercise
+>- When was the last time you published some data?
+>- What licence did you distribute it under?
+>- Where did you share the data?
+{: .discussion}
+
+Licences in Geoscience
+----------------------
+
+In general, data sources in geoscience such as CMIP5 and CMIP6 are available under FAIR licences. That is, you can freely use the data, but there are still some requirements. For example, if you base your research on CMIP6 data, you must give proper credit to the project if you deposit your research data somewhere. This is specified in the Terms of Use.
+
+You can find the Terms of Use for some widely used projects here:
+
+- CMIP6: <https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html>
+- CMIP5: <https://pcmdi.llnl.gov/mips/cmip5/terms-of-use.html>
+- CORDEX: <http://is-enes-data.github.io/cordex_terms_of_use.pdf>
+
+>## Exercise
+>- Where can you find the licence for CMIP6 data? What about CMIP5?
```

Comment:
```suggestion
```
That's mentioned just before the discussion box

Reply:
Perhaps add "Can you find additional terms of licences that are applicable to your situation?"